src/components/inscribete/FindVoterCenter/findYourCenterMachine.tsx

```diff
@@ -1,17 +1,15 @@
 import { createMachine, assign } from "xstate"
+
 import { FindYourCenterContext, FindYourCenterEvent } from "./types"
+import { VoterInformationResource } from "../../../packages/practica/resource"
 
 const isNumberExpr = new RegExp(/^\d+$/)
 
-const getVoterDetails = (voterId?: string) =>
-  fetch(`https://api.paravotar.org/consulta?voterId=${voterId}`).then(
-    response => {
-      if (!response.ok) {
-        throw new Error("HTTP status code: " + response.status)
-      }
-      return response.json()
-    }
-  )
+const getVoterDetails = async (voterId?: string) => {
```

Comment:
Async/Await seems unnecessary here if we are going to return a promise already

Reply:
done.